core/src/main/scala/org/apache/spark/serializer/SerializerManager.scala

```diff
@@ -70,6 +70,8 @@ private[spark] class SerializerManager(
   private[this] val compressRdds = conf.getBoolean("spark.rdd.compress", false)
   // Whether to compress shuffle output temporarily spilled to disk
   private[this] val compressShuffleSpill = conf.getBoolean("spark.shuffle.spill.compress", true)
+  // Size of the chunks to be used in the ChunkedByteBuffer
+  private[this] val chunkSizeMb = conf.getSizeAsMb("spark.memory.chunkSize", "4").toInt
```

Comment:
I think default needs to be "4m" and not 4, cf https://github.com/apache/spark/blob/529f847105fa8d98a5dc4d20955e4870df6bc1c5/core/src/main/scala/org/apache/spark/shuffle/BlockStoreShuffleReader.scala#L52

Reply:
Ok so apparently both will work, nevermind